### PR TITLE
Document serverset config

### PR DESCRIPTION
**Pull request by brian-brazil**
@fabxc

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -202,6 +206,32 @@ services:
 [ tag_separator: <string> | default = , ]
 ```
 
+### Zookeeper Serverset SD configurations `<serverset_sd_config>`
+
+Serverset SD configuration SD configurations allow retrieving scrape targets
```
Comment (fabxc) [inline content/docs/operating/configuration.md]: Double tap.

Comment (brian-brazil) [inline content/docs/operating/configuration.md]: Fixed.

File: content/docs/operating/configuration.md [hunk 2]
```diff
@@ -202,6 +206,32 @@ services:
...
+* `__meta_serverset_endpoint_port`: the port of the default endpoint
+* `__meta_serverset_endpoint_host_<endpoint>`: the host of the given endpoint
+* `__meta_serverset_endpoint_port_<endpoint>`: the port of the given endpoint
+* `__meta_serverset_status`: the status of the member
+
+```
+# The Zookeeper servers.
+servers: [ <host:post> ]
+# Path can point to a single serverset, or the root of a tree of serversets.
+path: <string>
```
Comment (fabxc) [inline content/docs/operating/configuration.md]: The correct field is `paths` as a list, right?
For servers `, ...` for list indication would be good. `post` -> `port`, though we just went with `<host>` as this is consistent with Go.

```
servers: [ <host>, ... ]
paths:
  [ - <string> ]
```

Comment (brian-brazil) [inline content/docs/operating/configuration.md]: Yip, guess I copied consul too closely. Fixed, thanks.

**Comment by fabxc** [comment]
:+1:

**Pull request merged**